src/mds/MDSDaemon.cc

```diff
@@ -149,6 +149,13 @@ void MDSDaemon::asok_command(
   if (command == "status") {
     dump_status(f);
     r = 0;
+  } else if (command == "sleep") {
```

Comment:
This is a great debugging tool. However, `sleep` might be a little too soft/generic for this command. How about `lock` or `stall`?

Reply:
how about lockup?